app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -242,7 +242,8 @@ public function edit($assetId = null)
 
         return view('hardware/edit', compact('item'))
             ->with('statuslabel_list', Helper::statusLabelList())
-            ->with('statuslabel_types', Helper::statusTypeList());
+            ->with('statuslabel_types', Helper::statusTypeList())
+            ->with('settings', Setting::getSettings());
```

Comment:
I think we have the settings in a shared view now so this isn't needed, no?

https://github.com/snipe/snipe-it/blob/8eb68d3901dd13e3e631fc4189537941e43bcd2a/app/Http/Middleware/AssetCountForSidebar.php#L29-L34